spec.md

```diff
@@ -0,0 +1,228 @@
+    A Registry Doc artifact (`application/vnd.oci.sample-repo-doc.config.v1`) which represents the contents of a repo cloud be represented as:
+    - `application/vnd.oci.sample-repo-doc.summary.layer.v1.md` - used as the readme of a repo, displayed on a registry webpage.
+    - `application/vnd.oci.sample-repo-doc.content.layer.v1.tar` - provides detailed documentation, including multiple `.md` files for offline reading.
+
+**Vendor|Org|Entity definitions** utilize the `layer.mediaType` with the following format:  
+`[registration-tree]`.`[org|company|entity]`.`[layerType]`.`[optional-layerSubType]`.layer.`[version]`.`[fileFormat]`+`[optional-compressionFormat]`
+
+- **`registration-tree`** - represents an IANA root tree. See [iana registration trees][iana-trees] for public and private tree options.
+- **`org|company|entity`** - represents an open source foundation (`oci`, `cncf`), a company or some unique entity.
+- **`layerType`** - a value representing the short name of the type.
+- **`optional-layerSubType`** - provides additional extensibility of a `layerType`
+- **`version`** - provides artifact authors the ability to revision their schemas and formatting, enabling tools to identify which format they will process. The content format (json, yaml) may not revision, but the content within the layer may evolve. Using a version enables artifact tools to identify the format for processing.
+- **`fileFormat`** - provides for standard, or custom formats, enabling artifact authors to leverage existing or custom content processing libraries.
+- **`optional-compressionFormat`** - in addition to the fileFormat, optional compression is used. (eg: `+gzip`, `+zstd`)
+
+### Example Layer Types
+
+| Artifact Type | mediaType |
+| - | - |
+| Simple text file layer | `application/text` |
+| Non-compressed OCI Image layer | `application/vnd.oci.image.layer.v1.tar` |
+| [gzip] Compressed OCI Image layer | `application/vnd.oci.image.layer.v1.tar+gzip` |
+| [gzip] Compressed Docker Image layer | `application/vnd.docker.image.rootfs.diff.tar+gzip` |
+| Non-compressed Helm Chart layer | `application/vnd.cncf.helm.chart.layer.v1.tar` |
+| [gzip] Compressed Helm Chart layer | `application/vnd.cncf.helm.chart.layer.v1.tar+gzip` |
+| Singularity SIF layer | `application/vnd.sylabs.sif.layer.v1.sif` |
+| Sample Doc Artifact Summary layer | `application/vnd.oci.sample-repo-doc.summary.layer.v1.md` |
+| Sample Doc Artifact Content layer| `application/vnd.oci.sample-repo-doc.content.layer.v1.tar` |
+
+## Optional: Defining Config Schema
+
+When defining an artifact type, the persistance of the artifact may be broken up into content and configuration. Configuration may be stored as a layer, or it may be stored and referenced by the `manifest.config`.
+
+While the value of `manifest.config.mediaType` is used to determine the artifact type, the persistance of a `config` object is OPTIONAL. Artifacts can push a null reference for `config.json` persistance.
```

Comment:
What is meant by "null reference for `config.json` persistance"  

sp on persistence